base-notebook/Dockerfile.ppc64le

```diff
@@ -0,0 +1,105 @@
+# Copyright (c) IBM Corporation 2016 
```

Comment:
OK, if you feel strongly against having Dockerfile.ppc64le in the repo, we will remove it. 

Regarding the license, I will need to check that and get back to you.